doc/src/proxies.md

```diff
@@ -0,0 +1,23 @@
+# Proxies
+
+`rustup` provides proxy wrapper through [components](https://github.com/rust-lang/rustup/blob/master/doc/src/concepts/components.md) and you can see the list of the proxies [here](https://github.com/rust-lang/rustup/blob/22dc71adbd7e19de0090f991b3609e5f14910dcb/src/lib.rs#L17-L28)
+
+- `rustc`: `rustc` is a proxy for the compiler for the Rust programming language, provided by the project itself and it matches `rustc` component.
+
+- `rustdoc`: `rustdoc` is a proxy for the standard Rust distribution ships with a tool called rustdoc that generates documentation for Rust projects and it matches `rustc` component.
```

Comment:
Again the language here is very complex, a simpler example could be:

```suggestion
- `rustdoc` is a tool distributed in the `rustc` component which helps you to generate documentation for Rust projects.
```

Reply:
Similar simplifications can be made to other entries.

In general, try not to be over-complex in your language or description, simpler is better at this point.